src/yardstick/cli/cli.py

```diff
@@ -28,13 +29,21 @@ def cli(ctx, verbose: bool, config_path: str):
     if verbose:
         log_level = "DEBUG"
 
+    class DeltaTimeFormatter(logging.Formatter):
+        def format(self, record):
+            duration = datetime.datetime.utcfromtimestamp(record.relativeCreated / 1000)
+            record.delta = duration.strftime("%H:%M:%S")
+            return super().format(record)
+
     logging.config.dictConfig(
         {
             "version": 1,
             "formatters": {
                 "standard": {
+                    "()": DeltaTimeFormatter,
                     # [%(module)s.%(funcName)s]
-                    "format": "%(asctime)s [%(levelname)s] %(message)s",
+                    # "format": "%(asctime)s [%(levelname)s] %(message)s",
```

Comment:
I can remove the line instead of commenting it out, but this format currently isn't exposed via configuration or anything, so someone would have to come and put the line back here to get timestamps back.